lib/identity_validations/service_provider_validation.rb

```diff
@@ -30,11 +30,11 @@ def self.included(base)
     #         we just enforce uniqueness, without whitespace.
     ISSUER_FORMAT_REGEXP = /\A[\S]+\z/.freeze
 
-    def redirect_uris_are_parsable
+    def redirect_uris_are_valid
       return if redirect_uris.blank?
 
       redirect_uris.each do |uri|
-        next if uri_valid?(uri) || uri_custom_scheme_only?(uri)
+        next if !uri.include?('*') && (uri_valid?(uri) || uri_custom_scheme_only?(uri))
```

Comment:
Wanted to use `uri.exclude?('*')` here but Ruby threw a Method Not Found error with "not_a_url".exclude?('*'). Not entirely sure why but also probably not worth the effort of looking into.

Reply:
`String#exclude?` is implemented in ActiveSupport and not core Ruby. Could `require 'active_support/core_ext/string/exclude'` or similar at the top of the file to make it available here, but it's a horse apiece to me.

Reply:
Meh, the ! doesn't bother me that much